src/EditorFeatures/Core/Implementation/InlineRename/CommandHandlers/RenameCommandHandler_SaveHandler.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using Microsoft.CodeAnalysis.Editor.Commands;
+using Microsoft.VisualStudio.Text.Editor;
+
+namespace Microsoft.CodeAnalysis.Editor.Implementation.InlineRename
+{
+    internal partial class RenameCommandHandler : ICommandHandler<SaveCommandArgs>
```

Comment:
If the file is already saved, then any active session will be committed when the EditAndContinueWorkspaceService transitions from Design mode to Run mode.